examples/producer_consumer.py

```diff
@@ -32,6 +32,7 @@
 
 loop = asyncio.get_event_loop()
 queue = asyncio.Queue(loop=loop)
-asyncio.ensure_future(produce(queue, 10), loop=loop)
```

Comment:
Yesterday, during AsyncIO workshop at PyCON-FR, it was hard for two attendees to understand why we have ensure_future for one coroutine, and not for the other.

I've proposed to change the example as proposed by @sylvainb, it seems more understandable by available beginners.
I've also suggested to propose a pull request for that.